www/~/%username/subscriptions.json.spt

```diff
@@ -0,0 +1,41 @@
+""" Get or change authenticated user's subscriptions.
+"""
+from aspen import Response
+from gratipay.exceptions import NoTeam
+
+[------------------------------------------------------------------------]
+
+if user.ANON:
+    raise Response(403, _("Please sign in first."))
+
+else:
+    participant = user.participant
+
+    if request.method == 'GET':
+        slug = request.qs.get("team_slug", "")
+
+        if slug: # Get subscription info only for this team.
+            try:
+                subscription = participant.get_payment_instruction(slug)
+            except NoTeam:
+                raise Response(400, _("Invalid team name."))
+            out = {
```

Comment:
We should maintain consistency in the format of what we return for the batch endpoint vs filtered endpoint. If we're returning 6 keys per item in the batch endpoint, we should be returning 6 keys per item in the filtered endpoint too... I think the best way to do this is to call `get_giving_for_profile` anyway, and filter the results in Python.